devel/distcc/Makefile

```
@@ -42,6 +46,10 @@ PLIST_FILES=	${_BIN_FILES:S|^|bin/|} \
 
 OPTIONS_DEFINE=		AVAHI CLANGLINK DOCS IPV6 LLVMLINK PUMP GUI
 
+.if ${FLAVOR:U} == pump
+OPTIONS_SET=	PUMP
```

I think this is more commonly used, and is less bug prone as it doesn't overwrite a variable.
```suggestion
OPTIONS_DEFAULT+=	PUMP
```